python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -204,6 +214,16 @@ def __init__(self, config, name, schema, metadata) -> None:
             flagDefs.add(schema.join(middleName, "flag_bigpsf"), ("The Gaussianized PSF is "
                                                                   "bigger than the aperture"
                                                                   ))
+        # PSF photometry
+        if self.config.doPsfPhotometry:
+            for sF in self.config.scalingFactors:
+                baseName = self.ConfigClass.getGaapResultName(sF, "PsfFlux", name)
+                baseString = f"with PSF aperture after scaling the seeing by {sF}"
```

Comment:
Might be worth specifying that it's a multiplicative scale factor to make it completely unambiguous.